src/resources/Notifications/notifications.ts

```diff
@@ -0,0 +1,13 @@
+import API from '../../APICore';
+import Resource from '../Resource';
+import Subscriptions from './Subscriptions/Subscriptions';
+
+export default class Notifications extends Resource {
+    subscriptions: Subscriptions;
```

Comment:
Will there be other resources than Subscriptions?

Reply:
for now there is none in the swagger and TBH I dont think so

Reply:
There is none implemented right now, and tbh I dont think so.

Reply:
[
![Screenshot from 2020-11-11 13-15-27](https://user-images.githubusercontent.com/45688129/98848683-15e7d380-2420-11eb-8546-af20ab82212d.png)
](url)

Reply:
I'm pretty confident we can put the calls directly in the Notifications resource